src/shared/leftSidebar/LeftSidebar.tsx

```diff
@@ -6,26 +6,35 @@ import {
   BsFillBookmarkFill,
   MdExplore,
 } from "../../assets/icons/icons";
+import { NavLink } from "react-router-dom";
 
 const LeftSidebar = () => {
   return (
     <aside className="leftSidebar-container">
       <ul className="leftSidebar-list-items list-style-none">
-        <li className="leftSidebar-list-item active font-bold">
-          <AiFillHome className="leftSidebar-list-icon" />
-          <a href="/" className="list-item-label">Home</a>
+        <li className="leftSidebar-list-item font-bold">
+          <NavLink to="/" className="flex-row align-center">
+            <AiFillHome className="leftSidebar-list-icon" />
+            <span className="list-item-label">Home</span>
+          </NavLink>
         </li>
```

Comment:
can map it in array of options as to follow ```DRY```

Reply:
Yes I will do later when I will add functionality.